htmlSanityCheck-core/src/test/groovy/org/aim42/htmlsanitycheck/check/BrokenHttpLinksCheckerSpec.groovy

```diff
@@ -4,14 +4,18 @@ import org.aim42.htmlsanitycheck.Configuration
 import org.aim42.htmlsanitycheck.collect.SingleCheckResults
 import org.aim42.htmlsanitycheck.html.HtmlConst
 import org.aim42.htmlsanitycheck.html.HtmlPage
-import org.aim42.htmlsanitycheck.tools.Web
+import org.aim42.htmlsanitycheck.test.dns.CustomHostNameResolver
 import org.wiremock.integrations.testcontainers.WireMockContainer
 import spock.lang.Ignore
 import spock.lang.IgnoreIf
 import spock.lang.Shared
 import spock.lang.Specification
 import spock.lang.Unroll
 
+import java.lang.reflect.Field
+import java.lang.reflect.Proxy
+
+
```

Comment:
Please do not make more than one new line behind a new block (I know this is not consequently done over all of the repository).